src/ReactWrapper.js

```diff
@@ -286,6 +286,17 @@ export default class ReactWrapper {
   }
 
   /**
+   * Returns the HTML of the node.
+   *
+   * NOTE: can only be called on a wrapper of a single node.
+   *
+   * @returns {String}
+   */
+  html() {
+    return this.single(n => findDOMNode(n).outerHTML.replace(/\sdata-reactid[^>]+/g, ''));
```

Comment:
My understanding is that <code>data-reactid</code> will always be appended by react at the end.  I can definitely dig into it a little more to make sure I'm correct.  I agree a more robust regex might be better in the long run though, in addition to another test case where <code>data-reactid</code> is somewhere else in the HTML.

Thanks for the feedback!  I should have an updated pull request sometime tonight.